### PR TITLE
CI for the homebrew assets

### PR DESCRIPTION
**Pull request by ds-sloth**
Looks like it's working! Please review and check that everything looks fine.

### CODE REVIEW

**Comment by Wohlstand** [comment]
Question: did you performed translations update?
I mean, do:
```bash
ASSETS_ROOT1="...path to assets root..."
cd .github/ci-helper
bash translate_update.sh "${ASSETS_ROOT}"
cd ../..
```
That script puts updated translations into assets, and applies patches are supplied at mainstream assets.

**Comment by Wohlstand** [comment]
At my desktop builds you may found the `name: Apply update to translations` step, that actually updates translations and applies patches (character names, etc.)

**Comment by ds-sloth** [comment]
I just added that now, thanks for the note!

File: .github/workflows/homebrew-assets-ci.yml [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+jobs:
+  build:
+    name: ${{ matrix.source.name }} for ${{ matrix.target.name }}
+    runs-on: ubuntu-latest
+    container: ${{ matrix.target.docker_image }}
+    env:
+        DEVKITPRO: /opt/devkitpro
+    strategy:
+      fail-fast: false
```
Comment (Wohlstand) [inline .github/workflows/homebrew-assets-ci.yml:26]: If this set to `false`, that means, if any job will fail, the chain will continue. That means, thing would turn into the mess if one job depends on a previous.

**Pull request merged**